hkmc2/shared/src/main/scala/hkmc2/semantics/flow/FlowAnalysis.scala

```diff
@@ -70,18 +82,17 @@ class FlowAnalysis(using tl: TraceLogger)(using Raise, State, Ctx):
         case stmt: DefineVar =>
           val rhs = typeProd(stmt.rhs)
           stmt.sym match
-          case sym: FlowSymbol =>
-            constrain(rhs, C.Flow(sym))
+          case sym: FlowSymbol => constrain(rhs, C.Flow(sym))
+          case _ => ()
```

Comment:
Is this the correct thing to do, or is it just an error-prone kludge (and maintenance burden) for a case that's not yet handled?